.github/ISSUE_TEMPLATE/new-package.md

```diff
@@ -2,7 +2,7 @@
 name: Request a new package
 about: Ask for a new package to be added to Fedora CoreOS
 title: 'New Package Request: <package name>'
-labels: ''
+labels: 'kind/enhancement'
```

Comment:
On that note, maybe we should have a new label for new package requests. Not blocking of course.